server/src/controllers/user_controller.ts

```diff
@@ -14,7 +15,7 @@ export class UserController implements interfaces.Controller {
         private readonly userService: UserService,
     ) {}
 
-    @Get("/me", extractJWTAuthMiddleware)
+    @Get("/me", extractJWTAuthMiddleware, accessRightsMiddleware([Role.USER, Role.DOCTOR]))
```

Comment:
You don't actually want to use it here. this route should be available to all logged in users